### PR TITLE
FYLE-2891u6y-create-statement

### PR DESCRIPTION
**Pull request by Kulsekar4**

### CODE REVIEW

File: src/admin/paths/admin@statements@create_card_transactions.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+post:
+  tags:
+    - Statements
+  summary: Create corporate card transactions out of the uploaded statement
+  description: |
+    This API helps in converting the statement with the mapping chosed and create card transactions out of the statement.
+  operationId: statements_prevew
```
Comment (modernwarfareuplink) [inline src/admin/paths/admin@statements@create_card_transactions.yaml:7]: _preview

File: src/components/schemas/statements.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtadfdsdf
+    statement_mapping:
+      type: object
+      properties:
+        description_column:
+          type: string
+          description: |
+            Specifies the header name of the description column in the statement.
```
Comment (modernwarfareuplink) [inline src/components/schemas/statements.yaml:14]: Pls use the same description as below (or) update all the below descriptions.

Ex: `Specifies the mapping of description column on the statement`

**Review by modernwarfareuplink — APPROVED** [review]
lgtm, have added minor comments. Pls fix and assign to next reviewer @Kulsekar4

File: src/components/schemas/statements.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
+preview_in:
```
Comment (aditya999123) [inline src/components/schemas/statements.yaml:1]: create_card_transaction_from_statement_in

Comment (aditya999123) [reply src/components/schemas/statements.yaml:1]: card_transaction_from_statement_in

**Pull request closed without merging**